sources/packages/web/src/views/student/financial-aid-application/ConfirmSubmission.vue

```diff
@@ -2,20 +2,27 @@
   <div class="p-component">
     <div class="p-card p-m-4">
       <Section title="Confirm Submission Page..." />
-      <FooterNavigator previous="financial-info" />
+      <FooterNavigator v-bind:previous="financialInfo" />
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import Section from "../../../components/generic/Section.vue";
 import FooterNavigator from "../../../components/generic/FooterNavigator.vue";
+import { routeConstants } from "../../../constants/routes/RouteConstants";
 
 export default {
   components: {
     Section,
     FooterNavigator,
   },
+  setup() {
+    const financialInfo = routeConstants.FINANCIALINFO;
```

Comment:
I would recommend renaming it to financialInfoRoute.

Reply:
done